data-cube-curation/values.yaml

```diff
@@ -0,0 +1,84 @@
+# Please fill those values.
+# Optional keys are commented out
+#
+# Some of those keys will fill a configmap while other will fill 3 secrets.
+# An existing secret can be used for each of those 3 sections.
+dataCubeCuration:
+  # baseURI: ""
+  # debug: ""
+  sparql:
+    existingSecret: null
+    # username: ""
+    # password: ""
+    endpoint: ""
+    # updateEndpoint: ""
+    readModelEndpoint: ""
+    # readModelUpdateEndpoint: ""
+    graphStoreEndpoint: ""
+    # graphStoreUsername: ""
+    # graphStorePassword: ""
+  s3:
+    existingSecret: null
+    accessKey: ""
+    secretKey: ""
+    bucket: ""
+    endpoint: ""
+  gitlab:
+    existingSecret: null
+    pipeline: ""
+    branch: ""
+    token: ""
+
+
+replicaCount: 1
+
+image:
+  repository: zazuko/datacube
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  runAsUser: 65534
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  # Enabling the ingress also sets a default baseURI
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: data-cube-curation.localhost
+      paths: [/]
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
```

Comment:
Is there an expected baseline expected for this app? If so please configure it as the default.

Reply:
I did that because the [default template](https://github.com/helm/helm/blob/fd00ca260a7132bc2f4678f1ff2f8253feac317c/pkg/chartutil/create.go#L140-L150) (when running `helm create [chart]`) has this comment on it:

```yaml
resources: {}
  # We usually recommend not to specify default resources and to leave this as a conscious
  # choice for the user. This also increases chances charts run on environments with little
  # resources, such as Minikube. If you do want to specify resources, uncomment the following
  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
  # limits:
  #   cpu: 100m
  #   memory: 128Mi
  # requests:
  #   cpu: 100m
  #   memory: 128Mi
```

Reply:
Ok, fine by me. I usually try to come up with a sensible baseline for applications I'm able to estimate.